Formula/flint.rb

```diff
@@ -0,0 +1,56 @@
+class Flint < Formula
+  desc "C library for number theory"
+  homepage "http://flintlib.org"
+  url "http://flintlib.org/flint-2.5.2.tar.gz"
+  sha256 "cbf1fe0034533c53c5c41761017065f85207a1b770483e98b2392315f6575e87"
+  head "https://github.com/wbhart/flint2.git", :branch => "trunk"
+
+  depends_on "gmp"
+  depends_on "mpfr"
+
+  def install
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  def caveats
+    <<~EOS
+      For C/C++ preprocessors to properly include flint headers you may need to set
```

Comment:
This doesn't seem specific to homebrew in which case it shouldn't be a caveat.

Reply:
Yeah, for a moment there I was thinking `opt_include` is Homebrew-specific and should be mentioned, but apparently users should just use `/usr/local/include/flint`. Will remove.